### ISSUE TITLE
plans to add table support?

### ISSUE THREAD

**Issue by giosifelis** [fouita/tailwind-editor#8]
Hello,  

Are there any plans to add table support?

**Comment by haynajjar** [comment]
Yes, but not at the moment, you can copy and paste tables from wherever you like (other webpage, xls, word...) and update the texts (as long as you don't change the style or hit enter inside the table)
